tensorflow/java/src/main/java/org/tensorflow/OperationBuilder.java

```diff
@@ -256,6 +256,20 @@ public OperationBuilder setAttr(String name, Shape value) {
     return this;
   }
 
+  public OperationBuilder setAttr(String name,  String[] value) {
+    Graph.Reference r = graph.ref();
+    try {
+      Object[] objects = new Object[value.length];
+      for (int i = 0; i < value.length; i++) {
+        objects[i] = value[i].getBytes(Charset.forName("UTF-8"));
+      }
+      setAttrStringList(unsafeNativeHandle, name, objects);
```

Comment:
Two things:
(1) Let's tighten the `try` block
(2) Let's avoid repeated `Charset` lookups.

So, something like:

```java
Charset utf8 = Charset.forName("UTF-8");
Object[] objects = new Object[value.length];
for (int i = 0; i < value.length; ++i) {
  objects[i] = value[i].getBytes(utf8);
}
Graph.Reference r = graph.ref();
try {
  setAttrStringList(unsafeNativeHandle, name, objects);
} finally {
  r.close();
}
```